CMakeLists.txt

```diff
@@ -31,52 +30,50 @@ if(${CMAKE_GENERATOR} STREQUAL "Ninja")
     endif()
 endif()
 
-# Default Blizzard Community Platform API key
-if (DEFINED SC_DEFAULT_APIKEY)
-    add_compile_options(-DSC_DEFAULT_APIKEY="${SC_DEFAULT_APIKEY}")
+# Install paths
+if (SC_USE_FLAT_INSTALL)
+  set(SIMC_INSTALL_BIN ".")
+  set(SIMC_INSTALL_SHARED ".")
+else()
+  set(SIMC_INSTALL_BIN "bin")
+  set(SIMC_INSTALL_SHARED "share/SimulationCraft/SimulationCraft")
 endif()
 
-# output a compile_commands.json used for editor tooling
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+# helper function to enable warnings on targets
+function(sc_enable_warnings target)
+  target_compile_options(${target} PRIVATE
+    $<$<CXX_COMPILER_ID:MSVC>:/W4>
```

Comment:
You *really* don't want to enable /W4, it's simply obnoxious.

Reply:
Seems level 3 is what we use in the handmade VS projects.